packages/runtime/container-runtime/src/runtimeLayerCompatState.ts

```diff
@@ -63,9 +63,12 @@ export const loaderSupportRequirements: ILayerCompatSupportRequirements = {
 export const runtimeCompatDetailsForDataStore: ILayerCompatDetails = {
 	...runtimeCoreCompatDetails,
 	/**
-	 * The features supported by the Runtime layer across the Runtime / Loader boundary.
+	 * The features supported by the Runtime layer across the ContainerRuntime / DataStoreRuntime boundary.
```

Comment:
```suggestion
	 * The features supported by the Runtime layer across the Runtime / DataStore boundary.
```

We are calling the layers `Runtime` and `DataStore`